project-repo/templates/common/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Report an issue
+description: Create a bug report to fix an issue
+title: ""
+labels: ""
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to report an issue and help improve PROJECT_TODO! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
+
+        By submitting an Issue to this repository, you agree to the terms within the
+        [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+
+        Be careful with sensitive information and security vulnerabilities. In order to report bugs that could contain
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/PROJECT_TODO/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug PROJECT_TODO` to collect relevant information.
+
+
+        FIXME: snap?
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      options:
+        - label: I have searched the existing issues and found none that matched mine
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is, including what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce it
+      description: >
+        Detail the steps taken to reproduce this error, what was expected, and whether this issue can be reproduced
+        consistently or if it is intermittent.
+      placeholder: |
+        Where applicable, please include:
+        * Code sample to reproduce the issue
+        * Log files (redact/remove sensitive information)
+        * Application settings (redact/remove sensitive information)
+        * Screenshots
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        Provide details about the environment you experienced the issue in.
+        * Distribution and version
+        * PROJECT_TODO version
+        * Desktop environment
+        * Hardware
+        * etc.
+
+        Ubuntu users can run `ubuntu-bug PROJECT_TODO --save=/tmp/report` and paste the file content bellow.
```

Comment:
typo: below*.

I would start the part with ubuntu user, as it’s going to collect those information above. So it’s either one or the other.

Also, the "below" had triple markdown backquote marked as raw type: the goal is that content of files and other pecial symbols are not marshalled or converted by mistake to markdown, please reuse this.

Reply:
I'm pretty there's a way to mark it as plain text so nothing gets formatted. I'll look into it.

Reply:
tell me if you find anything without the markup

Reply:
I already switched it. You can add a "parser" option to the validator section and then GitHub will recognize what's typed in there as if it was written in the specified language